src/Steps/Release/CreateProject.php

```diff
@@ -82,6 +82,16 @@ public function run(InputInterface $input, OutputInterface $output)
             throw new Exception("Could not create project");
         }
 
+        // Revert composer / recipe-core changes to composer.json
+        $library = $this->getProject();
+        $path = $library->getComposerPath();
+        $repo = $library->getRepository();
+        $status = $repo->run("status", [$path]);
+        if (stripos($status, 'Changes not staged for commit:')) {
+            $this->log($output, "Reverting installer changes to composer.json");
+            $repo->run("checkout", ["--", $path]);
+        }
```

Comment:
how come you're reverting? I don't quite follow this step

Reply:
recipe-module rewrites the root composer.json, but those changes must NOT be committed to source control.